sale_stock_available_to_promise_release/models/product_product.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    def _get_next_replenishment_date(self):
+        self.ensure_one()
+        picking_type_in = self.env.ref("stock.picking_type_in")
+        move_line = self.env["stock.move"].search(
+            [
+                ("product_id", "=", self.id),
+                ("date_expected", ">=", fields.Date.today()),
```

Comment:
could be expected in the past and not yet received. Then expected date is today

Reply:
So the filter should find everything that has not been received yet -> product = product, state not in cancel or done

Reply:
>= will select moves where date is today as well, then we're fine, right?